PMM-11278 QAN under the hood. (#1239)
percona/pmm-doc: docs/get-started/query-analytics.md

[SLOW_QUERY_LOG]: ../setting-up/client/mysql.md#slow-query-log
[PERFORMANCE_SCHEMA]: ../setting-up/client/mysql.md#performance-schema

## QAN under the hood

> To understand how metrics are calculated and details about QAN visit:

Reviewer: ```suggestion
To understand how metrics are calculated and to learn more details about QAN, see 

```

> [QAN under the hood](../details/query-analytics-details.md)

Reviewer: ```suggestion
[QAN under the hood](../details/query-analytics.md)
```

[SLOW_QUERY_LOG]: ../setting-up/client/mysql.md#slow-query-log
[PERFORMANCE_SCHEMA]: ../setting-up/client/mysql.md#performance-schema

## QAN under the hood

> To understand how metrics are calculated and to learn more details about QAN, see 

Reviewer: ```suggestion
To understand how metrics are calculated and to learn more details about QAN, see 
```

> [QAN under the hood](../details/query-analytics.md)

Reviewer: ```suggestion
[QAN under the hood](../details/query-analytics.md).
```
